grapher/stackedCharts/MarimekkoChart.tsx

```diff
@@ -75,7 +74,7 @@ import {
     LabelCandidateWithElement,
     MarimekkoBarProps,
 } from "./MarimekkoChartConstants"
-import { ErrorValue } from "../../coreTable/ErrorValues"
+import { findLastIndex } from "lodash"
```

Comment:
🚨 bundle size detective here! you know the drill

Reply:
Oh sorry, I thought this way it can do tree shaking (as opposed to import * from lodash).